server.js

```diff
@@ -138,10 +138,18 @@ function Iterator (down, req, encode) {
   this.batch = req.batch || 0
 
   if (req.options) {
-    if (req.options.gt === null) req.options.gt = undefined
-    if (req.options.gte === null) req.options.gte = undefined
-    if (req.options.lt === null) req.options.lt = undefined
-    if (req.options.lte === null) req.options.lte = undefined
+    if (!req.options.gt) {
```

Comment:
This excludes falsy yet valid range options. A better check is `== null`, although that might exclude range options that are valid in `bytewise`. So the best solution would be `if ('gt' in req.options)`, if we can make that work in the client-server protocol.